tests/Functional/Bundle/Extractor/PropertyTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Functional\Bundle\Extractor;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Zenstruck\Foundry\Bundle\Extractor\DoctrineTypes;
+use Zenstruck\Foundry\Bundle\Extractor\Property;
+
+class PropertyTest extends KernelTestCase
+{
+    /**
+     * @var Property|object|null
+     */
+    private $property;
+
+    protected function setUp(): void
+    {
+        self::bootKernel();
+        $em = static::getContainer()->get('doctrine.orm.default_entity_manager');
+        $this->property = new Property($em);
+    }
+
+    /**
+     * @before
+     */
+    public function skipIfNotUsingFoundryBundle(): void
```

Comment:
I don't _think_ this is required.

Reply:
ok remove lets try it will pass now. fingers crossed